tensorboard/plugins/projector/projector_plugin.py

```diff
@@ -422,6 +431,14 @@ def _read_latest_config_files(self, run_path_pairs):
                 with tf.io.gfile.GFile(config_fpath, "r") as f:
                     file_content = f.read()
                 text_format.Merge(file_content, config)
+            elif tf.io.gfile.exists(default_config_fpath):
+                with tf.io.gfile.GFile(default_config_fpath, "r") as f:
+                    file_content = f.read()
+                text_format.Merge(file_content, config)
+                for embedding in config.embeddings:
```

Comment:
I'm a little unclear about why this branch (the default root config) needs to expand embedding metadata paths to absolute here, whereas the other branch (the per-run config) doesn't need to do this.  Can you add some brief commentary about this?  E.g. is it because the paths are interpreted relative to the run path, or something like that?

Reply:
That's exactly it. It's interpreted relative to the run path. I'll add a comment explaining. :)